scheduler/scheduler-node/src/main/java/org/ow2/proactive/resourcemanager/task/client/RMNodeClient.java

```diff
@@ -111,6 +107,12 @@ public void renewSession() {
         }
     }
 
+    private void checkSessionNoNull() throws NotConnectedException {
+        if (sessionId == null) {
+            throw new NotConnectedException("You are not connected to RM. Call 'connect()' first.");
```

Comment:
You are not connected to the Resource Manager. Call 'connect()' first before using the interface.

Reply:
Ok